### PR TITLE
Several improvements/fixes to the schedule editor

### PR DESCRIPTION
**Pull request by terceiro**

### CODE REVIEW

File: wafer/static/css/wafer.css [hunk 1]
```diff
@@ -70,6 +70,19 @@ ul.profile-links li {
     background-color: white;
 }
 
+#wafer-edit-schedule-bucket .nav-tabs a.active {
+  font-weight: bold;
+}
```
Comment (stefanor) [inline wafer/static/css/wafer.css:75]: These tabs still look pretty awful. It'd be really nice to put them in a visual tab / block of some sort.

image

However, this is certainly an improvement, so won't block this PR on that.

File: wafer/schedule/templates/wafer.schedule/edit_schedule.html [hunk 2]
```diff
@@ -119,6 +119,9 @@ <h2>{% trans 'Bucket' %}</h2>
                     id="talk{{ talk.talk_id }}"
                     data-toggle="tooltip" data-placement="left"
                     title="{{ talk.title }}"
+                    {% if talk not in talks_unassigned %}
+                    hidden
```
Comment (stefanor) [inline wafer/schedule/templates/wafer.schedule/edit_schedule.html:123]: I think that should be indented, to match the rest of the template style

Comment (terceiro) [reply wafer/schedule/templates/wafer.schedule/edit_schedule.html]: yeah, fixed

File: wafer/static/css/wafer.css (hunk 1 above)
Comment (terceiro) [reply wafer/static/css/wafer.css:79]: fixed with some spacing.

bucket-nav

I tried to make it look like actual tabs, but I couldn't figure out how yet. I think a good chunk of that html needs to be rewritten.

Comment (stefanor) [reply wafer/static/css/wafer.css:79]: Probably :( And this sections below factored out, because there's a lot of duplication.

**Pull request merged**